python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -0,0 +1,1089 @@
+from __future__ import absolute_import, division, print_function
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+from collections import namedtuple
+import numpy as np
+
+# LSST imports
+from lsst.afw.geom import Point2D
+from lsst.afw.image import (ImageF, MaskedImageF, PARENT)
+import lsst.afw.detection as afw_det
+import lsst.meas.base as meas_base
+from lsst.afw.table import Point2DKey
+from lsst.pex.exceptions import LengthError
```

Comment:
LSST code tends to import this as `pexExcept` and then explicitly use `pexExcept.LengthError`. I also think it's cleaner to be explicit about where things are coming from in the code rather than importing things without context for the person reading the source.